theme/js/script.js

```diff
@@ -176,22 +204,104 @@ $.getUrlParameter = function(name){
 };
 
 function countCharacter() {
-    $('.text-area').on('input', function(){
+    $('.text-area').on('input', function () {
         var maxlength = $(this).attr('maxlength');
         var currentLength = $(this).val().length;
         var charCounter = $('.contact-form-container .character-counter');
 
-        if( currentLength >= maxlength ){
+        if (currentLength >= maxlength) {
             // If the current length of the text is 500 or more characters, display the specified text.
             charCounter.text('You have reached the maximum number of characters.');
-        } else{
+        } else {
             // If the length of the text is less than 500 characters, display the number of characters remaining.
             charCounter.text(maxlength - currentLength + ' characters remaining');
         }
 
-        if( currentLength >= 250) {
+        if (currentLength >= 250) {
             // Add class "limit-warning" to the character counter when there are 50 characters remaining or less.
             charCounter.addClass('limit-warning');
         }
     });
 }
+
+function openDonationTab() {
+
+    setInitialPosition = function(){
+        // Defines the variable for the position of the first of the donate form buttons.
+        var initialPosition = $('.form-buttons.active button').first().position();
+        // Defines the position of the middle of each of the donate sum buttons.
+        var buffer = $('.form-buttons.active .donate-sum').outerWidth() / 2;
+
+        // sets the initial position of the arrow to equal the middle of the first button.
+        $('.donation-content-panel .triangle-left').css({left: initialPosition.left + buffer});
+    };
+
+    setInitialPosition();
+
+    $('.tab .tablinks').click(function () {
+        //Clicking on tablinks adds an active class and removes it from the sibling element.
+        $(this).addClass('active').siblings().removeClass('active');
+        if ($(this).hasClass('monthly')) {
```

Comment:
This section can be refactored give me a shout for guidance.

Reply:
done